src/UI/Buyer/src/app/components/checkout/checkout-address/checkout-address.component.html

```diff
@@ -8,6 +8,39 @@
   >
     <small>Loading Shipping Selections...</small>
   </ngx-spinner>
+  <div *ngIf="!isAnon" class="form-group">
+    <label for="selectBuyerLocation" translate
+      >CHECKOUT.CHECKOUT_ADDRESS.BUYER_LOCATION</label
+    >
+    <fa-icon
+      class="ml-2"
+      placement="right"
+      ngbTooltip="Select the buyer location this order will be attributed to"
+      [openDelay]="500"
+      [icon]="faQuestionCircle"
+    ></fa-icon>
+    <select
+      [ngModel]="selectedBuyerLocation?.ID"
+      (ngModelChange)="onBuyerLocationChange($event)"
+      class="custom-select"
+      id="selectBuyerLocation"
+    >
+      <option [ngValue]="null" default disabled translate>
+        CHECKOUT.CHECKOUT_ADDRESS.SELECT_BUYER_LOCATION
+      </option>
+      <option
+        *ngFor="let location of existingBuyerLocations?.Items"
+        [ngValue]="location.ID"
+      >
+        <span class="font-weight-bold"
+          >{{ location.Street1
+          }}<span *ngIf="location.Street2"> {{ location.Street2 }}</span
+          >, {{ location.City }} {{ location.State }} {{ location.Zip }}</span
+        >
```

Comment:
Here, maybe it help for clarity-sake to utilize the location's name vs. it's street address?  Or maybe a combination of both.  That way the user is able to select the location based off of quick name association vs. having to know the address?

Reply:
Just updated